Signal/src/ViewControllers/MessageDetailViewController.swift

```diff
@@ -703,6 +706,11 @@ class MessageDetailViewController: OWSViewController, MediaGalleryDataSourceDele
         // no - op
     }
 
+    @objc func didTapSent(sender: UIGestureRecognizer) {
+        let messageTimestamp = "\(message.timestamp)"
+        UIPasteboard.general.string = messageTimestamp
```

Comment:
I think we should be a little more careful about clobbering someone's pasteboard. A 'tap' can easily happen inadvertently.

Since this is a debug feature, maybe we should make it like 2 taps with two fingers or something.

Reply:
I made it a long press.